docs/api_doc/discretisation/index.rst

```diff
@@ -17,6 +17,7 @@ into continuous intervals.
 :class:`EqualWidthDiscretiser()`         Sorts values into intervals of equal size.
 :class:`ArbitraryDiscretiser()`          Sorts values into intervals predefined by the user.
 :class:`DecisionTreeDiscretiser()`       Replaces values by predictions of a decision tree, which are discrete
+:class:`IncreasingWidthDiscretiser()`    Sorts variable into increasing width intervals
```

Comment:
following the slides you linked to the class, would it be not better to say: `sorts variable into geometrical intervals`?